bindata/network/ovn-kubernetes/ipsec.yaml

```diff
@@ -27,6 +27,13 @@ spec:
         openshift.io/component: network
         kubernetes.io/os: "linux"
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: network.operator.openshift.io/dpu-host
```

Comment:
Does `dpu-host` mean any host with a DPU, or the literal DPU itself?

Reply:
The former, any host with a DPU.

Reply:
aha, makes sense.